packages/cli/src/AbstractServer.ts

```diff
@@ -70,6 +75,10 @@ export abstract class AbstractServer {
 		this.endpointWebhook = config.getEnv('endpoints.webhook');
 		this.endpointWebhookTest = config.getEnv('endpoints.webhookTest');
 		this.endpointWebhookWaiting = config.getEnv('endpoints.webhookWaiting');
+
+		this.serverId = generateHostInstanceId(instanceType);
```

Comment:
should we move this to `start.ts` and `webhook.ts` commands to be consistent with `worker.ts`.